lib/controllr.rb

```diff
@@ -10,8 +11,19 @@ def contractor_count
   end
 
   def user_count(employment)
-    data = fetch("/users.json?&iDisplayLength=1000&search[active_in][]=true&search[employment_in][]=#{employment}")
-    data.fetch('iTotalDisplayRecords', 0)
+    data = user_data.select { |user| user['employment'] == employment }
+    data.length
+  end
+
+  def average_age
+    data = user_data.select { |user| user['employment'] == 'employee' }
+
+    ages = data.map { |user|
+      date_of_birth = Date.parse(user['date_of_birth'])
+      Age.from_date(date_of_birth)
+    }.compact
+
+    ages.inject(&:+) / ages.length
```

Comment:
we don't have activesupport, and I don't think we should add it for those few use cases we have.